package.json

```diff
@@ -80,21 +80,22 @@
     "lint:production:stylelint": "tsc --noCheck --outDir ./dist && stylelint '**/*.styles.ts' --custom-syntax postcss-lit --no-color",
     "test": "per-env",
     "test:development": "pnpm test:development:web-test-runner",
-    "test:development:playwright": "playwright test --quiet --ui-port 6007 --update-snapshots --update-source-method=overwrite",
+    "test:development:playwright": "http-server ./dist/playwright/coverage --silent & playwright test --quiet --ui-port 6007 --update-snapshots --update-source-method=overwrite",
     "test:development:playwright:update": "rimraf ./src/aria-snapshots && playwright test --quiet --project aria --update-snapshots --update-source-method=overwrite --workers 100%",
     "test:development:vitest": "vitest --config ./vitest.config.js & chokidar './src/eslint/**' './src/stylelint/**' --ignore './src/eslint/rules/*.test.ts' './src/stylelint/*.test.ts' --initial --silent --command 'tsc --noCheck --outDir ./dist'",
     "test:development:vitest:comment": "Excluded from `test:development` because Vitest muddies the console when running alongside Web Test Runner.",
     "test:development:web-test-runner": "web-test-runner --watch",
     "test:production": "pnpm start:production:esbuild && npm-run-all --aggregate-output --print-label --parallel test:production:*",
     "test:production:playwright:aria": "playwright test --quiet --only-changed origin/main",
+    "test:production:playwright:components": "playwright test --project components",
     "test:production:playwright:visuals": "playwright test --ignore-snapshots --quiet --only-changed origin/main",
     "test:production:playwright:visuals:comment": "Snapshots are ignored because this is meant to be only a quick smoke test telling us when we've written a test incorrectly.",
     "test:production:vitest": "tsc --noCheck --outDir ./dist && vitest run --config ./vitest.config.js",
     "test:production:web-test-runner": "web-test-runner",
     "typecheck": "per-env",
     "typecheck:development": "tsc --erasableSyntaxOnly --outDir ./dist -w",
     "typecheck:production": "tsc --erasableSyntaxOnly --outDir ./dist",
-    "postinstall": "is-ci || pnpm dlx playwright@1.52.0 install --only-shell --with-deps chromium",
+    "postinstall": "is-ci || pnpm dlx playwright@1.52.0 install --only-shell --with-deps chromium firefox webkit",
```

Comment:
As Glide Core becomes more widely used, it makes sense to expand testing to other browsers. I've been putting this off because:

1. Firefox and Safari don't work out of the box with Web Test Runner. The issue is likely on our end. 
2. Web Test Runner doesn't support intra-test parallelization or sharding. So we'd have to run separate test job for each browser. Otherwise, our CI times would blow up.

Playwright, on the other hand, is able run our tests (at least the few I added in this PR) out of the box without issue. And it supports parallelization of all kinds, so the time cost of testing all three browsers in CI should be negligible.